JDBC/DatabasesPractice/src/main/java/com/laba/solvd/database/localDBS/Objects.xml

```diff
@@ -0,0 +1,123 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Just a suggestion - The conventional location for non-java file is in resource folder.